app/service/FileService.scala

```diff
@@ -63,4 +77,15 @@ class FileService @Inject()(connector: BindingTariffFilestoreConnector) {
     r => PublishedFileAttachment(r.id, r.fileName, r.mimeType, size)
   }
 
+  private val maxFileSize = 10485760
+  private val allowedTypes = Set(
```

Comment:
I would define these extensions in the config file (`application.conf`) and then read it in memory using `AppConfig.scala`.
This would allow us to have different allowed types for each environment (local, QA, production).
Also, if in the future we want to remove or add a file type, we would just need to change the config

Reply:
That is a good idea.  Will change